lib/driver.js

```diff
@@ -83,6 +84,10 @@ class AndroidDriver extends BaseDriver {
     // start UiAutomator
     this.bootstrap = new Bootstrap(DEVICE_PORT, this.opts.websocket);
     await this.bootstrap.start(this.opts.appPackage, this.opts.disableAndroidWatchers);
+    // handling unexpected shutdown
+    this.bootstrap.onUnexpectedShutdown.catch((err) => {
+      this.startUnexpectedShutdown(err);
```

Comment:
don't you need to set `this.bootstrap` to null here as well, since it's shut down at this point?